### DISCUSSION TITLE
Unable to read selected option when DBeaver set with Theme=Dark

### DISCUSSION THREAD

**Discussion by emanueol** [dbeaver/dbeaver#36622] [Q&A] [answered]
### Description

1. Simple, just start DBeaver > Windows > Preferences > any option clicked on left panel is black on black..

My dbeaver settings:
DBeaver > Windows > Preferences > User Interface > Appearance:
Theme= Dark
Color and Font theme= Classic Theme (current)
![IMG_20241220_201018](https://github.com/user-attachments/assets/7ecc8b95-bf24-4445-95c9-0296d02b4e79)
![IMG_20241220_201052](https://github.com/user-attachments/assets/a95e6f5f-252e-4f7d-9720-b2d10969dbc7)

Windows:
Doesnt matter if windows in Light or Dark mode.

### DBeaver Version
DBeaver Enterprise 21.3.0.202112052011

### Operating System
Windows 11 Enterprise

### Steps to reproduce

1. Simple, just start DBeaver > Windows > Preferences
2. And any option clicked on left panel is black on black..

### Additional context
1l Above photos may look bit readable, because probably of phone photo ai pos processing or something, because on my screen i cant read.. the dark blue background its like black and foreground color is black.

2) In addition to somehow fix this, and because this is enterprise edition in my job/work place, please feel free to share the name of those 2 color settings so i manually change them.

Thanks in advance ~

**Comment by serge-rider** [comment, marked as answer]
This bug is already fixed.
You use 3-years old DBeaver, could you try a fresh one?

**Comment by emanueol** [comment]
> This bug is already fixed. You use 3-years old DBeaver, could you try a fresh one?

Oh I see, whats the minimum version where this was fixed and whats latest version?
Enterprise Edition.

Thanks ~

**Comment by serge-rider** [comment]
You could try 22.0 but I am not 100% sure..

**Reply by emanueol** [reply]
Seems will not be easy or at least take more time than I wished to have company get use latest version (will try asap) but meanwhile id there any easy workaround?
I basically installed EE because of ODBC support that would be nice eventually connect to other data sources that i see theres respective odbc driver in company software portfolio.

2. Additionally, when I installed CE and during creation of Snowflake connection Dbeaver automatically downloaded the Snowflake jdbc driver, p}but for example when tried create connection to an impala db, the jdbc wasn't downloaded automatically (so it errored with missing driver). Any suggestion in why this happened and how can i get/install jdbc driver manually. Whats right place to download and extract ?

**Comment by emanueol** [comment]
Hi, I tested now in community version 25.1.2 and it's fixed the left options no longer black on dark theme.

But now the problem is that the "Preview" is still hard to read:
windows > preferences > User Interface > appearance > Colors and Fonts > search for "font"
![image](https://github.com/user-attachments/assets/feb30b47-3260-4981-9358-adaeadf7aad8)

Is this still a bug, or is there any them color I can set to change that preview from black into some color more readable?

Thanks ~
